### DISCUSSION TITLE
Shelly Plus PM Mini support?

### DISCUSSION THREAD

**Discussion by greymda** [hoylabs/OpenDTU-OnBattery#791] [Q&A]
Hello community,
I am looking into options to have my consumption measured so I can make use of the "Zero export" option as my local regulations prohibit any export to the grid.

I stumbled across this unit Shelly Plus PM Mini. So it's a power meter with 16A support, link to official site https://www.shelly.com/en/products/shop/shelly-plus-pm-mini

Can someone please advice if this is supported by the project? I did use search but no results for this particular model.

All the best!

**Comment by spcqike** [comment]
It should be supported. You can define your powermeter as HTTP(S)+JSON. So you need to use your Shelly’s uri and get the total power meter reading. (Don’t know right know, the wiki contains a lot of info for Shelly 3EM, I guess ts kind of similar) 

just make sure to uncheck „inverter is behind powermeter“ (as I guess it isn’t in your case)

afterwards the DPL will try to match your Shelly’s power reading.

**Reply by greymda** [reply]
thank you

**Comment by Manos1966** [comment]
Hi @greymda 

I need to ask something out of curiosity: How i.e. WHERE do you want to place the Shelly Plus PM Mini ?

I thought you want to ensure that "no export to the grid takes place".
- This implies, you need to monitor your Electricity Meter
- With a Shelly plus PM mini you would need to PHYSICALLY connect it between your Electricity Meter and your house 

Which is DEFINITIVELY NOT what your should be doing....

For such things, you should only have Shelly 3em in mind because this product monitors consumption CONTACTLESS i.e. there is no physical intervention to the cables of your Electricity Meter.

Just my 2cents...

**Reply by spcqike** [reply]
> because this product monitors consumption CONTACTLESS

you still need fuses and a physical connection to every line, to measure the voltage accordingly. 

But a Shelly 1pm mini wouldn’t be enough for your household. Of course :) it’s just for a single breaker circuit / device.
